cmd/backup/config.go

```diff
@@ -45,6 +45,13 @@ type Config struct {
 	WebdavPath                 string        `split_words:"true" default:"/"`
 	WebdavUsername             string        `split_words:"true"`
 	WebdavPassword             string        `split_words:"true"`
+	SSHHostName                string        `split_words:"true"`
+	SSHPort                    string        `split_words:"true" default:"22"`
+	SSHUser                    string        `split_words:"true"`
+	SSHPassword                string        `split_words:"true"`
+	SSHIdentityFile            string        `split_words:"true" default:"/root/.ssh/id_rsa"`
```

Comment:
I wonder if we can skip this option and mandate mounting the key file at `/root/.ssh/id`. This could possibly only conflict with someone extending the image, but used as-is I cannot think of any scenario where this would become a problem right now. Any opinion on this?

Reply:
If in any circumstances this will be a problem or conflict with someone, the default path can be changed with this environment. I can't think of any case right now

Reply:
Should we make the default `/root/.ssh/id` then? It'd feel weird to map a non-RSA key (ED25519 for example) to a file called like this. people might even thing it HAS to be a RSA key?

Reply:
The devil is in the details. `man ssh` tells me:

```
     -i identity_file
             Selects a file from which the identity (private key) for public key authentication is read.  The default is ~/.ssh/id_dsa, ~/.ssh/id_ecdsa, ~/.ssh/id_ecdsa_sk, ~/.ssh/id_ed25519,
             ~/.ssh/id_ed25519_sk and ~/.ssh/id_rsa.  Identity files may also be specified on a per-host basis in the configuration file.  It is possible to have multiple -i options (and multiple
             identities specified in configuration files).  If no certificates have been explicitly specified by the CertificateFile directive, ssh will also try to load certificate information
             from the filename obtained by appending -cert.pub to identity filenames.
```

Maybe we can leave `id_rsa` as the default, but add a note to the documentation about how non-RSA key algos are supported as well?

Reply:
You are right and i didnt read that. I changed to .ssh/id and updated documentation.

Reply:
Ok, then I'll change it myself real quick before cutting the release, sorry for the back and forth.